contracts/rust/deployer/src/builder.rs

```diff
@@ -60,8 +69,27 @@ impl DeployerArgs {
             },
             Contract::EspTokenProxy => {
                 let token_recipient = self.token_recipient.unwrap_or(admin);
-                let addr =
-                    crate::deploy_token_proxy(provider, contracts, admin, token_recipient).await?;
+                let token_name = self
+                    .token_name
+                    .clone()
+                    .unwrap_or_else(|| "Espresso".to_string());
```

Comment:
If you're not doing much "work" in the `or` case, you can use `unwrap_or("Espresso".to_string())`

Reply:
https://github.com/EspressoSystems/espresso-network/commit/0b3500890d58b76b74789003acaf63cffbb84703